.gitignore

```diff
@@ -18,3 +18,12 @@ __pycache__/
 
 # Ignore vim's .swp files
 .*.swp
+
+# Elastic Beanstalk Files
+.elasticbeanstalk/*
+!.elasticbeanstalk/*.cfg.yml
+!.elasticbeanstalk/*.global.yml
```

Comment:
Had never seen this `!` in the .gitignore before, had to look it up. This is super minor, but if I'm understanding things correctly [here](https://git-scm.com/docs/gitignore), it doesn't appear that the `.elasticbeanstalk/*.cfg.yml` file can be included since its parent directory is discluded from the repo. Maybe we delete these lines?